File: sample/dracula.java

// Import statement
import java.util.Arrays;

// Class declaration
public class Test {

    // Static variable
    static int staticVar = 10;

    // Instance variables
    final String instanceVar;
    final int[] arrayVar;

    // Constructor
    public Test(String instanceVar, int[] arrayVar) {
        this.instanceVar = instanceVar;
        this.arrayVar = arrayVar;
    }

    // Static method
    public static void staticMethod() {
        System.out.println("This is a static method.");
    }

    // Instance method
    public void instanceMethod(param1, param2) {
        System.out.println("This is an instance method.");
        System.out.println("Instance variable: " + this.instanceVar);
        System.out.println("Array variable: " + Arrays.toString(this.arrayVar));
    }

    // Main method
    public static void main(String[] args) {
        // Local variable
        String localVar = "Hello, World!";

        // Array
        int[] array = { 1, 2, 3, 4, 5 };

        // Object instantiation
        Test demo = new Test(localVar, array);

        // Method calls
        staticMethod();
        demo.instanceMethod();

        // Control structures
        if (staticVar > 5) {
            System.out.println("Static variable is greater than 5.");
        } else {
            System.out.println("Static variable is not greater than 5.");
        }

        for (int i = 0; i < array.length; i++) {
            System.out.println("Array element " + i + ": " + array[i]);
        }

        while (staticVar < 15) {
            System.out.println("Incrementing static variable.");
            staticVar++;
        }

        try {
            System.out.println("Trying to divide by zero...");
            int result = 10 / 0;
            System.out.println(result);
        } catch (ArithmeticException e) {
            System.out.println("Caught an exception: " + e.getMessage());
        } finally {
            System.out.println("This is the finally block.");
        }
    }
}

File: sample/dracula.py
import pandas as pd
class SyntaxDemo:
    # Class variable
    class_var = 10

    def __init__(self, instance_var, array_var):
        # Instance variables
        self.instance_var = instance_var
        self.array_var = array_var

    @staticmethod
    def static_method():
        print("This is a static method.")

    def instance_method(self):
        print("This is an instance method.")
        print("Instance variable: ", self.instance_var)
        print("Array variable: ", self.array_var)

# Local variable
local_var = "Hello, World!"

# List
array = [1, 2, 3, 4, 5]

# Object instantiation
demo = SyntaxDemo(local_var, array)

# Method calls
SyntaxDemo.static_method()
demo.instance_method()

# Control structures
if SyntaxDemo.class_var > 5:
    print("Class variable is greater than 5.")
else:
    print("Class variable is not greater than 5.")

for i in array:
    print("Array element: ", i)

while SyntaxDemo.class_var < 15:
    print("Incrementing class variable.")
    SyntaxDemo.class_var += 1

try:
    print("Trying to divide by zero...")
    result = 10 / 0
except ZeroDivisionError as e:
    print("Caught an exception: ", e)
finally:
    print("This is the finally block.")